src/pkg/control/options.go

```diff
@@ -48,44 +47,50 @@ func Defaults() Options {
 }
 
 // ---------------------------------------------------------------------------
-// Restore Item Collision Policy
+// Restore Configuration
 // ---------------------------------------------------------------------------
 
-// CollisionPolicy describes how the datalayer behaves in case of a collision.
-type CollisionPolicy int
-
-//go:generate stringer -type=CollisionPolicy
 const (
-	Unknown CollisionPolicy = iota
-	Copy
-	Skip
-	Replace
+	defaultRestoreLocation = "Corso_Restore_"
 )
 
-// ---------------------------------------------------------------------------
-// Restore Destination
-// ---------------------------------------------------------------------------
+// CollisionPolicy describes how the datalayer behaves in case of a collision.
+type CollisionPolicy string
 
 const (
-	defaultRestoreLocation = "Corso_Restore_"
+	Unknown CollisionPolicy = ""
```

Comment:
do we ever plan on printing the restore policy variable? If we do it seems like this could cause confusion because all the others will have strings that represent the const but this will be empty

Reply:
I don't think it's unreasonable to equate empty to unset/unknown.

Reply:
I'm still a little hesitant about this. Having an exported field be a string where the the empty value is not a valid option (it really shouldn't be) can be error prone. If callers don't realize there's a function to generate a default set of options or if they want some non-default set of options it becomes easy to forget to set the field to something useful. At least with ints it's possible to associate a meaningful default to the field

Reply:
`Unknown = iota` is functionally no different.  In the abstract, both cases rely on the zero value to denote an unspecified behavior, and can hit that result in all the same ways.  Likewise, the interpretation of a zero-value int to be something meaningful a naming convention, not something related to its value.  Ultimately, it's all in the handling.

Reply:
>interpretation of a zero-value int to be something meaningful a naming convention, not something related to its value

I kind of have to push back on this point. Because strings are often printed as-is, having an empty string be associated with some defined behavior that matches a non-empty string description (i.e. default is "skip" but we interpret "" as "skip") doesn't make sense. Ints kind of escape that by having a layer of indirection between the value/behavior association and the printed name. Thus it's much easier and less confusing to associate 0 (or the zero-value) with a meaningful behavior

It also is something associated with the value, not the naming convention. By defining a set of iota "enums" (since golang doesn't technically have enums) one is associated behavior with specific values. The naming conventions certainly help humans determine which behavior is associated with each value, but one could just as easily pass in (possibly type cast) ints or name all the enums something like `enumVal0`, `enumVal1`... and the code would continue to work

Reply:
> i.e. default is "skip" but we interpret "" as "skip") doesn't make sense.

Fair, I'll give you that one.  In a nitpicky way, we're _falling back_ to skip, not equating empty to skip.  But that argument doesn't clarify the confusion as pointed out.  Its a matter of communication around default behavior and expectations, yeah?

On the other hand, printing iota stringers is only helpful you get to leverage the stringer interface.  In cases of serialization you're left with json like `"{ typeEnum: 3 }"`, and that's not helpful for human evaluation even when properly assigned.

> but one could just as easily pass in (possibly type cast) ints or name all the enums something like enumVal0, enumVal1... and the code would continue to work

Yeah, but one could do that with strings just as easily.  In neither case does the underlying type guarantee usage protections.  It's all "which of these matches in the switch statement". 

I think what I'm taking out of this is that, independent of the underlying type, we want to control for default assignment within structs like this, so that even if the caller passes in a non-standard value, we at least set the property hold a value that matches our expectations.